src/Container/test/ContainerStylesSpec.tsx

```diff
@@ -13,6 +13,7 @@ describe('Container styles', () => {
         <span>Title</span>
       </Container>
     );
-    assert.equal(getStyle(getDOMNode(instanceRef.current), 'display'), 'flex');
+    const dom = instanceRef.current as Element;
+    assert.equal(getStyle(dom, 'display'), 'flex');
```

Comment:
Use `expect` instead of `assert` as the test assertion to maintain code style consistency.

```js
 expect(container.firstChild).to.have.style('display', 'flex');
```